src/lib/events.ts

```diff
@@ -197,6 +198,7 @@ const mentionCommands: MentionCommand[] = [
 
 					if (user.cl.has(item.id)) icons.push(Emoji.CollectionLog);
 					if (user.bank.has(item.id)) icons.push(Emoji.Bank);
+					if ((sacrificedBank as ItemBank)[item.id]) icons.push(Emoji.Incinerator);
```

Comment:
Please fix the warning associated with your change. 

Perhaps:
```js
if ((sacrificedBank as ItemBank)[item.id] ?? 0 > 0) icons.push(Emoji.Incinerator);
```

Reply:
Threw the same warning, I believe my new commit solves the issue.